src/ypd/servlets/user_servlet.py

```diff
@@ -31,14 +109,16 @@ def logout(self):
     @route('/signup/', methods=['POST', 'GET'])
     def signup(self):
         form = RegistrationForm()
-        if request.method == 'POST' and form.validate_on_submit:
+        if form.validate_on_submit:        
```

Comment:
We need the `request.method == 'POST'` portion to prevent getting error messages before we try to submit the form

Reply:
The request.method should not be necessary. Form.validate_on_submit does the same thing. It works fine for me

Reply:
Yea, it shouldn't be necessary. But Flask-WTF sucks. This is what I see when I visit the signup page for the first time
![Screenshot from 2020-04-17 11-25-10](https://user-images.githubusercontent.com/46609378/79585735-3922ee00-809e-11ea-8ae7-083e3f1cee5d.png)